### PR TITLE
Update Niv, Nix shell hook, and gitignore

### PR DESCRIPTION
**Pull request by expede**
Screen Shot 2021-04-21 at 11 41 34 AM

...the same will happen with `lorri`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,209 @@
 /docs
 /node_modules
 /yarn-error.log
+
```
Comment (expede) [inline .gitignore:6]: 👇 Protip: https://gitignore.io

File: shell.nix [hunk 2]
```diff
@@ -6,9 +6,12 @@ in
...
   buildInputs = [
-    pkgs.just
     unstable.yarn
     unstable.nodejs-15_x
     unstable.niv
   ];
+
+  shellHook = ''
+    ${unstable.yarn}/bin/yarn install
+  '';
```
Comment (expede) [inline shell.nix:16]: The main event

**Comment by icidasset** [comment]
Same comment as in https://github.com/fission-suite/auth-lobby/pull/68 regarding gitignore file. Looks good otherwise 👍

**Comment by expede** [comment]
Looking at offline support: this one doesn't have the problem from the auth-lobby because no `just` action deleting stuff

**Review by icidasset — APPROVED** [review]

**Pull request merged**